packages/styleguide/src/components/Overlay/docs.md

```diff
@@ -110,3 +110,21 @@ responsive: Desktop small
   </OverlayBody>
 </OverlayRenderer>
 ```
+
+## Overlay `mini`
+
+```react
+noSource: true
+plain: true
+responsive: Mobile
+span: 2
+---
+  <OverlayRenderer mini isVisible onClose={() => {}} mini>
+    <OverlayToolbar title='Mini Overlay' onClose={() => undefined} />
```

Comment:
On mobile, it looks like a call out now.

Other call outs do not have a close button.

Do you think, we should hide it?

<img width="399" alt="Bildschirmfoto 2022-12-19 um 19 21 33" src="https://user-images.githubusercontent.com/331800/208493938-bfdb9b48-c766-46c5-a2d0-2e0d82211c46.png">